components/brave_shields/resources/panel/components/main-panel/hooks.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import getPanelBrowserAPI, { UIHandlerReceiver } from '../../api/panel_browser_api'
+
+export function useGetShieldsData() {
+  const [url, setUrl] = React.useState("")
+  const [totalBlockedResources, setTotalBlockedResources] = React.useState(0)
+
+  React.useEffect(() => {
+    const uiHandlerReceiver = new UIHandlerReceiver({
+      onSiteDataChanged: (currentSite) => {
+        setUrl(currentSite.location.url)
```

Comment:
@petemill eventually the url should be cleaned up so ui only shows host name or we can do the clean up on the backend?

Reply:
seems like the f/e only needs the site name, so get that as low down as possible, i.e. in the shields backend